lib/ask/runtime/channel_broker.ex

```diff
@@ -439,7 +439,23 @@ defmodule Ask.Runtime.ChannelBroker do
     DateTime.compare(not_after, Ask.SystemTime.time().now) != :gt
   end
 
+  defp log_contact(status, respondent) do
+    session = respondent.session |> Ask.Runtime.Session.load()
+    SurveyLogger.log(
+      respondent.survey_id,
+      session.current_mode.channel.type,
```

Comment:
I'm surprised we still don't have a function that receives a mode and returns its label (`SMS`/`IVR`/`Mobile Web`). We should use that here to avoid breaking the current interaction file format.